bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -286,6 +300,29 @@ def cancel(self):
         """Cancels the iterator, closing the stream."""
         self._response_iterator.cancel()
 
+    def create_retry_request(self):
```

Comment:
should all these new methods be private?

Reply:
In either case, they need docstrings.

Reply:
Done.